example.html

```diff
@@ -1,57 +1,58 @@
-            return "Sunrise";
-        } else if (prayer == adhan.Prayer.Dhuhr) {
-            return "Dhuhr";
-        } else if (prayer == adhan.Prayer.Asr) {
-            return "Asr";
-        } else if (prayer == adhan.Prayer.Maghrib) {
-            return "Maghrib";
-        } else if (prayer == adhan.Prayer.Isha) {
-            return "Isha";
-        } else if (prayer == adhan.Prayer.None) {
-            return "None";
+        function prayerName(prayer) {
+            if (prayer == adhan.Prayer.Fajr) {
+                return "Fajr";
+            } else if (prayer == adhan.Prayer.Sunrise) {
+                return "Sunrise";
+            } else if (prayer == adhan.Prayer.Dhuhr) {
+                return "Dhuhr";
+            } else if (prayer == adhan.Prayer.Asr) {
+                return "Asr";
+            } else if (prayer == adhan.Prayer.Maghrib) {
+                return "Maghrib";
+            } else if (prayer == adhan.Prayer.Isha) {
+                return "Isha";
+            } else if (prayer == adhan.Prayer.None) {
+                return "None";
+            }
         }
-    }
 
-    const coordinates = new adhan.Coordinates(66.7222444, 17.7189);
-    const params = adhan.CalculationMethod.MuslimWorldLeague();
-    params.polarCircleResolution = adhan.PolarCircleResolution.AqrabYaum;
-    params.highLatitudeRule = adhan.HighLatitudeRule.SeventhOfTheNight;
-    const date = new Date(2020, 05, 21);
-    
-    var prayerTimes = new adhan.PrayerTimes(coordinates, date, params);
+        const coordinates = new adhan.Coordinates(66.7222444, 17.7189);
+        const params = adhan.CalculationMethod.MuslimWorldLeague();
+        params.polarCircleResolution = adhan.PolarCircleResolution.AqrabYaum;
+        params.highLatitudeRule = adhan.HighLatitudeRule.SeventhOfTheNight;
+        const date = new Date(2020, 05, 21);
+
+        var prayerTimes = new adhan.PrayerTimes(coordinates, date, params);
 
-    var sunnahTimes = new adhan.SunnahTimes(prayerTimes);
+        var sunnahTimes = new adhan.SunnahTimes(prayerTimes);
 
-    document.write('Raleigh, NC - Muslim World League\n');
-    document.write(`Prayer times for ${moment(date).format('MMMM DD, YYYY')}\n`);
-    document.write(`Fajr: ${moment(prayerTimes.fajr).tz('Europe/Stockholm').format('MMMM DD, YYYY h:mm A')}\n`);
-    document.write(`Sunrise: ${moment(prayerTimes.sunrise).tz('Europe/Stockholm').format('MMMM DD, YYYY h:mm A')}\n`);
-    document.write(`Dhuhr: ${moment(prayerTimes.dhuhr).tz('Europe/Stockholm').format('MMMM DD, YYYY h:mm A')}\n`);
-    document.write(`Asr: ${moment(prayerTimes.asr).tz('Europe/Stockholm').format('MMMM DD, YYYY h:mm A')}\n`);
-    document.write(`Maghrib: ${moment(prayerTimes.maghrib).tz('Europe/Stockholm').format('MMMM DD, YYYY h:mm A')}\n`);
-    document.write(`Isha: ${moment(prayerTimes.isha).tz('Europe/Stockholm').format('MMMM DD, YYYY h:mm A')}\n`);
+        document.write('Raleigh, NC - Muslim World League\n');
```

Comment:
minor thing, but it looks like i changed the coordinates at some point but forgot to update the name here.

Reply:
what should it be now

Reply:
middle of nowhere, sweden??